app/controllers/twitter_sign_in.js

```diff
@@ -4,7 +4,7 @@ var oauth = require('../../lib/oauth.js')
 var logger = require('../../lib/logger.js')()
 
 var oauthRequestTokenHandler = function (req, res) {
-  return function (err, oauth_token, oauth_token_secret, results) {
```

Comment:
We have few js files also in bin. When I run
```
./node_modules/.bin/standard ./**/*.js --verbose | ./node_modules/.bin/snazzy
```

I'm getting

```

/home/magul/Dokumenty/kdp/streetmix/bin/compute_street_sizes.js
  14:7   error  Identifier 'date1_ms' is not in camel case       camelcase
  15:7   error  Identifier 'date2_ms' is not in camel case       camelcase
  18:7   error  Identifier 'difference_ms' is not in camel case  camelcase
  18:32  error  Identifier 'date1_ms' is not in camel case       camelcase
  18:43  error  Identifier 'date2_ms' is not in camel case       camelcase
  21:21  error  Identifier 'difference_ms' is not in camel case  camelcase
  23:1   error  Block must not be padded by blank lines          padded-blocks
  37:1   error  Block must not be padded by blank lines          padded-blocks

/home/magul/Dokumenty/kdp/streetmix/bin/download_translations.js
   6:15  error  Use path.join                 ) or path.resolve() instead of + to create paths. (no-path-concat
   9:7   error  Use path.join                 ) or path.resolve() instead of + to create paths. (no-path-concat
  16:19  error  Use path.join                 ) or path.resolve() instead of + to create paths. (no-path-concat
  20:22  error  Expected error to be handled  handle-callback-err

/home/magul/Dokumenty/kdp/streetmix/config/default.js
  2:15  error  Use path.join  ) or path.resolve() instead of + to create paths. (no-path-concat
  6:7   error  Use path.join  ) or path.resolve() instead of + to create paths. (no-path-concat

/home/magul/Dokumenty/kdp/streetmix/lib/transifex.js
   7:46  error  Identifier 'err_cb' is not in camel case   camelcase
   7:54  error  Identifier 'succ_cb' is not in camel case  camelcase
  33:1   error  Block must not be padded by blank lines    padded-blocks

✖ 17 problems
```

Don't we want to fix them also?